setup.py

```diff
@@ -46,6 +44,6 @@
     zip_safe=False,
     setup_requires=["katversion"],
     use_katversion=True,
-    install_requires=["pyephem", "katpoint", "matplotlib", "numpy", "pyyaml"],
+    install_requires=["astropy", "pyephem", "katpoint", "matplotlib", "numpy", "pyyaml"],
```

Comment:
I see `astropy` is only used in the notebooks, not any of the scripts or astrokat library.  Do the notebooks on COLAB need this?

Reply:
Indeed, this snuck in from another devel branch. Later versions will move from `ephem` to `astropy`, but it does not belong here.
I removed the requirement, this only muddies the branch